src/Exception/RequestEntityTooLargeException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Elastica\Exception;
+
+class RequestEntityTooLargeException extends \RuntimeException implements ExceptionInterface
+{
+    /**
+     * @var string
+     */
+    protected $message = 'Request entity is too large.';
```

Comment:
We should instead provide our own constructor:
```php
public function __construct(\Throwable $previous = null) {
    parent:__construct('Request entity is too large', 0, $previous);
}
```

WDYT?

Reply:
I dont mind - 
refactored